.zshrc

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this file needed? Looks like a local shell file, not something needed for the repository

Reply:
yeah dont know where this came from, will remove in the next commit